Source/Modules/main.cxx

```diff
@@ -965,6 +969,11 @@ int SWIG_main(int argc, char *argv[], const TargetLanguageModule *tlm) {
     }
   }
 
+  if (!cparse_ignore_attrs) {
+    Preprocessor_define("__has_cpp_attribut 201803L", 0);
```

Comment:
Typo: `__has_cpp_attribute`

But this seems wrongly defined - the expected usage is e.g. `__has_cpp_attribute(noreturn)` so your definition would mean:

```
#if __has_cpp_attribute(noreturn)
```

would expand to:

```
#if 201803L(noreturn)
```

Reply:
Yes, I will probably remove it, as this would be too complex to support.